src/actions/vstudio/vs2010_vcxproj.lua

```diff
@@ -1231,7 +1231,15 @@
 
 
 	function m.generateDebugInformation(cfg)
-		m.element("GenerateDebugInformation", nil, tostring(cfg.flags.Symbols ~= nil))
+		if cfg.flags.Symbols and _ACTION >= "vs2015" then
+			if cfg.flags.FastLink then
+				m.element('GenerateDebugInformation', nil, "DebugFastLink")
+			else
+				m.element('GenerateDebugInformation', nil, "Debug")
```

Comment:
The change is due to VS2015 having 3 valid options, while everything prior supported `true`/`false`, VS2015 supports `No`, `Debug` and `DebugFastLink`. The default value is `Debug`, so there should be support here to specify `No`. I agree with what @starkos was saying, this should be a proper API but I'm not sure if `symbols` is the correct name?